Modules/Messages/src/Messages.Infrastructure/Persistence/Database/Repository/MessagesRepository.cs

```diff
@@ -36,19 +36,14 @@ public async Task<Message> Find(MessageId id, IdentityAddress address, Cancellat
             .WithSenderOrRecipient(address)
             .FirstWithId(id, cancellationToken);
 
-        if (fillBody)
-        {
-            await FillBody(message);
-        }
+        await FillBody(new DbPaginationResult<Message>(new List<Message>() { message }, 1));
```

Comment:
Hmm...
Why are you building a pagniation result object, just to call the FillBody method, which itself just unwraps the message and passes it to the other FillBody method? 😮